backend/zfs_operations/services/pool_service.py

```
@@ -813,4 +813,67 @@ async def _parse_pool_history(self, output: str) -> Result[List[Dict[str, Any]],
             return Result.failure(PoolException(
                 f"Failed to parse pool history: {str(e)}",
                 error_code="POOL_HISTORY_PARSE_FAILED"
-            )) 
\ No newline at end of file
+            ))
+    
+    # === Additional methods for router compatibility ===
+    
+    async def get_zfs_iostat(self, 
+                           pools: Optional[List[str]] = None,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)